Add examples for popular platforms regarding LoadBalancer service exposing (#1601)
scylladb/scylla-operator: docs/source/exposing.md

On platforms with support for external load balancers, this Service provisions one. 
The accessibility of this load balancer's address depends on the platform and any customizations made; in some cases it may be reachable from the internal network or public Internet.

> Customizations are usually managed via Service annotations, key-values provided in `annotations` field are merged into each Service object.

Reviewer: Although the options imply it, it might be worth to add a note that they should configure it in passthrough mode and not redirect. (For other platforms and better background.)

> Customizations are usually managed via Service annotations, key-values provided in `annotations` field are merged into each Service object.

Reviewer 2: ```suggestion
Customizations are usually managed via Service annotations, key-value pairs provided in `annotations` field are merged into each Service object.
```

Author: done

> For example, to expose LoadBalancer only to internal network use following annotations:

Reviewer 2: ```suggestion
For example, to expose LoadBalancer only to internal network use the following annotations:
```

> EKS:

Reviewer 2: ![image](https://github.com/scylladb/scylla-operator/assets/32443348/d5521613-7f88-4fda-9e93-3edb4467d010)
Imo looks sort of ugly, maybe you could use bullets?
![image](https://github.com/scylladb/scylla-operator/assets/32443348/ef454141-7f70-4ea8-8d67-ebeee6c95acf)

Author: thanks, I added new extension which adds tabs:
![Selection_051](https://github.com/scylladb/scylla-operator/assets/3951334/64525a69-9009-490e-96c2-33701916a06f)

Reviewer 2: Nice! Might be useful for other stuff as well, we could use the dropdowns for outputs or manifests.

```yaml
apiVersion: scylla.scylladb.com/v1
kind: ScyllaCluster
spec:
  exposeOptions:
    nodeService:
     type: LoadBalancer
     annotations:
       service.beta.kubernetes.io/aws-load-balancer-scheme: internal
       service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
```

GKE:
```yaml
apiVersion: scylla.scylladb.com/v1
kind: ScyllaCluster
spec:
  exposeOptions:
    nodeService:
     type: LoadBalancer
     annotations:
       networking.gke.io/load-balancer-type: Internal
```

> Check platform documentation regarding LoadBalancer configuration to learn more about available options.

Reviewer 2: ```suggestion
Check platform-specific documentation regarding LoadBalancer configuration to learn more about available options.
```
nit

Author: done
